src/AspNetCore/aspnetcore_schema.xml

```diff
@@ -38,5 +38,12 @@
                 <attribute name="value" type="string" required="true"/>
             </collection>
         </element>
+
+        <element name="appSettings">
```

Comment:
People are going to confuse this with the top level app settings. https://msdn.microsoft.com/en-us/library/ms228154(v=vs.90).aspx Any reson not to use that one?

Reply:
That one is usually used by application.  We want to reserve one for ANCM only so that in the future we don't need to update schema file and redeploy aspnetcore.dll. only need to update the requesthandler assembly.

Reply:
I understand, but it should get a unique name.

Reply:
moduleSettings?

Reply:
moduleSettings, handlerSettings, settings...

Reply:
handlerSettings is better

Reply:
changed to handlerSettings